stories/action-sheet/action-sheet.stories.js

```diff
@@ -0,0 +1,198 @@
+import '../../dist/avatar.css';
+import '../../dist/icon.css';
+import '../../dist/action-sheet.css';
+
+export default {
+    title: 'Components/Action Sheet',
+    parameters: {
+        description: `
+An action sheet consists of a list of options a user can select from to complete an action. Actions can be
+clustered if there is not enough space on the screen
+            
+## Usage
+
+**Use the action sheet if:**
+
+- You need an option that provides more than one action.
+- It is really important that the user stays in context on a phone.
+- You only have a small number of actions.
+
+**Do not use the action sheet if:**
+
+- The menu provides only one option. In this case, consider using a button instead.
+- You need to show a hierarchical menu. In this case, use the menu button instead.
+- Your users would benefit more from a split button, which offers an easily-accessible default action, with the option to include additional actions.
+
+## Responsiveness
+
+The action sheet is fully responsive. On smartphones, the actions are displayed as a list inside a dialog.
+On tablets and desktop devices, the actions are displayed in a popover.
+        `,
+        tags: ['a11y', 'f3', 'theme']
+    }
+};
+
+export const actionSheetDesktop = () => `    
+<div class="fd-popover">
+    <div class="fd-popover__control">
+        <button class="fd-button fd-button--compact fd-button--transparent" 
+                aria-label="More Options" aria-controls="actionSheetDesktop" aria-expanded="false" 
+                aria-haspopup="true" onclick="onPopoverClick('actionSheetDesktop');">
+            <i class="sap-icon--settings"></i>
+        </button>
+    </div>
+    <div class="fd-popover__body" aria-hidden="true" id="actionSheetDesktop">
+        <ul class="fd-action-sheet fd-action-sheet--compact">
```

Comment:
```suggestion
        <ul class="fd-action-sheet fd-action-sheet--compact" aria-label="List of contextual options">
```

Reply:
I completely forgot about arias and roles. Thanks for noticing